src/bin/edje/edje_cc.c

```diff
@@ -195,6 +197,11 @@ main(int argc, char **argv)
           {
              eina_log_domain_level_set("edje_cc", EINA_LOG_LEVEL_INFO);
           }
+        else if (!strcmp(argv[i], "-in-tree"))
+          {
+             putenv("EFL_RUN_IN_TREE=1");
+             in_tree = 1;
```

Comment:
Maybe we could try a slightly different approach: we rely on `in_tree` global variable (which could be renamed to `efl_run_in_tree`) and, in the start of `main`, check for `getenv(EFL_RUN_IN_TREE)` is not 0 and if that's the case then set `efl_run_in_tree` to 1.

Reply:
Sounds better for me too.

Reply:
Nice, I'll try that.